components/match2/wikis/rainbow_six/match_summary.lua

```diff
@@ -404,8 +404,29 @@ end
 
 function CustomMatchSummary._createHeader(match)
 	local header = MatchSummary.Header()
-	header  :left(CustomMatchSummary._createOpponent(match.opponents[1], 'left'))
-			:right(CustomMatchSummary._createOpponent(match.opponents[2], 'right'))
+	header  :left(
+				mw.html.create(nil)
```

Comment:
Why `nil`?

Make this `'div'` instead. Then set this code up like this:

`brkts-popup-header-opponent` (this div) should have properties:

```
display: flex;
justify-content: flex-end; (for the right side we'll need flex-start)
```

Remove the width calc.

Reply:
Perfect, exactly what I needed